react/week1/todolist/src/todos.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const TodoItems = (props) => {
+    return (
+        <h2>
+            {props.activity},{props.duration}
+        </h2>
+    )
+}
+
+const RenderedTodos = (props) => {
```

Comment:
When passing props, you can actually use a very cool javascript feature called destructuring. For example

    TodoListRendering({activity, deadline})

Then you can access the variables without using `props.`